openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -162,14 +166,14 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 	}
 
```

Comment:
a few lines up from here, every time we get an error in the loop, please print it there, nested properly

Reply:
I think this error covers it okay. Isn't it?
https://github.com/openshift/etcd/pull/36/files#diff-ed50358308058964703579e6ea5d3cb9R158

Reply:
ah, didn't realize I did that.